garrysmod/lua/vgui/dhtml.lua

```diff
@@ -92,8 +92,13 @@ function PANEL:ConsoleMessage( msg )
 
 	end
 
+	local values = { ... }
+
+	-- If the first parameter is an object/array/function/undefined, just print this useless message
+	values[ 1 ] = values[ 1 ] or "*js variable*"

```

Comment:
Maybe do this to all values? this might also not play well with null or undefined. I know that it's gmod's default behavior already, but we could fix it!